sdk/communication/azure-communication-callautomation/tests/call_automation_automated_live_test_base.py

```diff
@@ -0,0 +1,94 @@
+import os
+import json
+import time
+from datetime import datetime, timedelta
+from typing import Any, Dict, Optional, Type
+
+import requests
```

Comment:
I wonder if we should keep similar imports together or having them separated as you have. If we keep them separated, should we add comments on why?

Reply:
I was following the convention as described in the PEP 8 Python style guide
<img width="606" alt="image" src="https://user-images.githubusercontent.com/89811028/229647143-ddc533e7-5591-48a1-a541-a3a0fed3b089.png">
https://peps.python.org/pep-0008/